customer-churn/steps/visualizer.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from zenml.integrations.facets.visualizers.facet_statistics_visualizer import (
+    FacetStatisticsVisualizer,
+)
+from zenml.repository import Repository
+
+
+def visualize_statistics():
+    """
+    This function will get the last pipeline run, get the last step of the pipeline run, and then
+    visualize the statistics of the last step.
+    """
+    try:
+        repo = Repository()
+        pipe = repo.get_pipelines()[-1]
```

Comment:
Personally I feel this is not the most reliable way of fetching pipelines. mybe pass the pipeline name as a parameter to this function and then fetch the pipeline by name with ..get_pipeline(pipeline_name=...)

Reply:
Sure

Reply:
I think this hasn't been resolved with this latest change..

Reply:
The reason why I am not using alexej suggested method because I have to StepContext as of now and I don't want to convert my visualisers in steps as it will run after pipeline runs completed, so for that reason I am using the usual one.

Reply:
not sure I understand why the StepContext prevents using of get_pipeline(pipeline_name=...) - lets chat about this :)

Reply:
As said If I use StepContext then I have to convert my visualizers as step, so I just want traditional python functions that's why like.